tests/e2e/script_test.go

```diff
@@ -130,6 +130,11 @@ func TestE2EScript(t *testing.T) {
 						exportResource = nil
 						shouldGetKubeObject = false
 
+					case "SLEEP":
```

Comment:
Adding timer in test always worries me. Is the purpose of this SLEEP to guarantee single http request log? My concern is mostly about  flakiness and test running latencies. 
For this case, I think 2 seconds are fine, but maybe we shall have some constraints on adding timer in test. Any suggestions?

Reply:
I think I can replace this with a magic annotation.  What I'm not sure about is whether this should be 1.118 or whether we want to live with the potentially flaky test and fix it in 1.119.  There will be code changes in the non-test path to detect the magic annotation.

Reply:
I'll vote to leave non-test change to 1.119.